src/debugging/python/PythonDebugHelper.ts

```diff
@@ -125,6 +127,8 @@ export class PythonDebugHelper implements DebugHelper {
         switch (projectType) {
             case 'django':
                 return 'Starting development server at (https?://\\S+|[0-9]+)';
+            case 'fastapi':
+                return '(Listening at: (https?://\\S+|[0-9]+) \(113392\))|(Uvicorn running on (https?://\\S+|[0-9]+) \(Press CTRL+C to quit\))' // HEY!
```

Comment:
This can be cleaned up :sweat_smile:

On the debug, we use `uvicorn`, which is the second (afte the `|`) but on production the first one (before the `|`).

The number `113392` is actually the PID of the master process. I was wondering if I need to match the whole line that identifies that the server is running. :thinking: Like: `Listening at: (https?://\\S+|[0-9]+) \(\d+\)` would be fine?

Reply:
I'm pretty sure it doesn't need to be the full line, but the code does expect only one group, with that group containing the URL or the port only. Maybe something like this (which uses non-capturing groups for the extra text):

`(?:Listening at: (https?://\\S+|[0-9]+))|(?:Uvicorn running on (https?://\\S+|[0-9]+)`

Reply:
Perfect. Thank you!

Reply:
Applied.